### DISCUSSION TITLE
How do I use symbols for `mode`

### DISCUSSION THREAD

**Discussion by luiz00martins** [nvim-lualine/lualine.nvim#642] [Q&A] [answered]
Instead of showing `NORMAL` or `INSERT` in the `mode` section, I want to display some icons, or a single letter (`N`, `I`, `V`, etc...).

Is that possible? If so, how do I set it up?

**Comment by shadmansaleh** [comment, marked as answer]
Use a custom component as shown [here](https://github.com/nvim-lualine/lualine.nvim/issues/614#issuecomment-1072275099)
